Source/debug.cpp

```diff
@@ -294,6 +295,84 @@ std::string DebugCmdLoadMap(const string_view parameter)
 	return "Welcome to this unique place.";
 }
 
+void WriteLE16(std::ofstream &out, uint16_t val)
+{
+	const uint16_t littleEndian = SDL_SwapLE16(val);
+	char data[2];
+	memcpy(data, &littleEndian, 2);
+	out.write(data, 2);
+}
+
+std::string ExportDun(const string_view parameter)
+{
+	std::ofstream dunFile;
+
+	std::string levelName = fmt::format("{}-{}.dun", currlevel, glSeedTbl[currlevel]);
+
+	dunFile.open(levelName, std::ios::out | std::ios::app | std::ios::binary);
+
+	WriteLE16(dunFile, DMAXX);
+	WriteLE16(dunFile, DMAXY);
+
+	/** Tiles. */
+	for (int y = 0; y < DMAXY; y++) {
+		for (int x = 0; x < DMAXX; x++) {
+			WriteLE16(dunFile, dungeon[x][y]);
+		}
+	}
+
+	/** Padding */
+	for (int y = 16; y < MAXDUNY - 16; y++) {
+		for (int x = 16; x < MAXDUNX - 16; x++) {
+			WriteLE16(dunFile, 0);
+		}
+	}
+
+	/** Monsters */
+	for (int y = 16; y < MAXDUNY - 16; y++) {
+		for (int x = 16; x < MAXDUNX - 16; x++) {
+			uint16_t monsterId = 0;
+			int mid = abs(dMonster[x][y]) - 1;
+			if (mid != -1) {

```

Comment:
This would write a moving monster twice and duplicates the monster in the dun file or? 
(one for mid positive and one for mid negative).
Do we want this?